src/User32/User32+MENUINFO.cs

```diff
@@ -15,9 +15,7 @@ public partial class User32
         /// Contains information about a menu.
         /// </summary>
 #if NETPORTABLE
```

Comment:
Empty conditional here, raise the question if we still want `Unicode` or not (I believe not as the windows versions needing it are long gone)

Reply:
Ya, declaring `Unicode` anywhere is deprecated, IMO. Thanks for catching this though, I used regex to remove them all, which didn't bother to tell me what it left behind. :)